code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -67,6 +67,8 @@
 
 	var/mob/living/simple_animal/hostile/spawner/nest
 
+	var/sentience_type = SENTIENCE_ORGANIC // Sentience type, for slime potions

```

Comment:
I wanted to expand this to human mobs in some other pr. This is mostly for future use, for example away boss monsters and such.